### PR TITLE
Attempt to explain for..in variable leak

### PR DESCRIPTION
**Pull request by KaoruDev**
@kytrinyx feel free to fix my grammar. Something I'm ok not understanding lol

### CODE REVIEW

File: comments/ruby/for_loop/for_loop.md [hunk 1]
```diff
@@ -19,4 +27,14 @@ end
...
 ```
 
+`Enumerable` methods do not affect surounding variables.
+
+```ruby
+person = 'alice'
+["alice", "bob", "charlie"].each do |person|
+  # your code
+end
+person #=> 'charlie'
```
Comment (kytrinyx) [inline comments/ruby/for_loop/for_loop.md]: This should be `person #= 'alice'`, right?

**Comment by kytrinyx** [comment]
This looks good, thanks! (Nothing wrong with your English :))

File: comments/ruby/for_loop/for_loop.md (hunk 1 above)
Comment (KaoruDev) [inline comments/ruby/for_loop/for_loop.md]: ack yeah nice catch!

**Comment by kytrinyx** [comment]
Thanks! I'll deploy this when I get home today.

**Pull request merged**